### DISCUSSION TITLE
How to make accent color button?

### DISCUSSION THREAD

**Discussion by DmitryBorodiy** [amwx/FluentAvalonia#567] [Q&A] [answered]
How I can make button accent in FluentAvalonia? Something like in UWP AccentButtonStyle.

**Comment by robloo** [comment, marked as answer]
FluentAvalonia has this style already built in. Just set `Classes="accent"` on any `Button`.

https://github.com/amwx/FluentAvalonia/blob/d9a80edfed14ef0ca18dd7e31f727abd90031797/src/FluentAvalonia/Styling/ControlThemes/BasicControls/ButtonStyles.axaml#L82-L107

Avalonia's styling system is quite different in some areas and this more closely follows the CSS way of doing things.

**Reply by DmitryBorodiy** [reply]
Thanks!
